samples/WinFormsDx12/Program.cs

```diff
@@ -0,0 +1,23 @@
+using System;
```

Comment:
All of the files need a header similar to `// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.`